Dockerfile

```diff
@@ -48,20 +48,17 @@ RUN yum install -y \
         openssl-devel \
         jq \
         python-devel \
-        python-pip \
-        yum clean all
+        python-pip && \
+    yum clean all
 
 # Install latest stable xrootd 4.12.5 version and its dependencies
 # hadolint ignore=DL3033
```

Comment:
This `hadolint ignore` rule seems to be not needed any longer since all the versions are pinned

Reply:
Updated. I have actually joined it now with the previous `yum install` command, so that we don't have to do unnecessarily `yum clean all` twice.